dist/formats/email_alert_signup/frontend/schema.json

```diff
@@ -55,10 +55,8 @@
     },
     "details": {
       "type": "object",
-      "additionalProperties": false,
```

Comment:
Had intended this as a temporary workaround while we're dropping `tags` but on second glance your suggestion below re: leaving tags in for now is obviously much more sensible. Updating...